packages/svelte-ux/src/routes/docs/utils/format/+page.svelte

```diff
@@ -83,13 +102,277 @@
   <div>{format(0.5678, 'percent', { locales: 'fr', fractionDigits: 1 })}</div>
 </Preview>
 
-<h2>Period formats</h2>
+<h1>Dates</h1>
+
+<h2>No format</h2>
+
+<Preview>
+  {format(myDate)}
+  <!-- TODO JYC?
+    before: 30/03/1982 00:00:00
+    after: Tue Mar 30 1982 00:00:00 GMT+0200 (heure d’été d’Europe centrale)
+  -->
```

Comment:
I'm not sure what to do here. It's maybe the "only" difference with `dateDisplay`!
I would remove this example. 
Ok with that?

Reply:
We can remove the example as the output is the same as `date.toString()`.

![image](https://github.com/techniq/svelte-ux/assets/177476/16e3f145-3682-4879-98b0-677a6b50bc5d)

It was nice to have a quick way to show a consolidated date/time, but not a big deal.  ATM we don't handle any time formatting, although I used `dateDisplay` for one example on LayerChart

![image](https://github.com/techniq/svelte-ux/assets/177476/636fc9a6-7900-4cc1-b2c1-cbac51706942)

I also use date-fns's `format()` for a few other time formats.

![image](https://github.com/techniq/svelte-ux/assets/177476/506782c6-4cb6-446b-8956-c1c1b0ffd128)

I figure it would be nice to have an intl. way to handle time as well.  Should we add `PeriodType.DayTime` (or `DateTime`) and `PeriodType.TimeOnly` (or just `Time`) with short and long variants?

- `PeriodType.DayTime`
  - short: 'M/d/yyyy hh:mm a'
  - long: 'M/d/yyyy hh:mm:ss a'
- `PeriodType.TimeOnly`
  - short: 'hh:mm a'
  - long: 'hh:mm:ss a'

Although this brings up a few time specific questions?
- Should we show as 24 hour or 12 hour am/pm.  I figure this is locale specific in most cases (US is am/pm typically)
- Do we only show seconds for `long`?  what about milliseconds (`SSS`).  Probably just handle via `custom`, but would be good to support it if possible (when I look at the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat), I don't see any mention of milliseconds, so maybe not possible... hmm.  If not, using date-fns for these use cases is fine.

Reply:
Great points. Wanted to push this out for later... But in the end, it's pretty easy to handle all these cases!

With `custom` you can already do all that, but yes, let's add `DayTime` & `TimeOnly`.
`intl` supports also `24 & am/pm` if I recall well. I will check.

Reply:
It does via `hour12` (I saw when I was looking for millisecond support, which I didn't see)

Reply:
All Done 🎉

(I resolve this one, let's open new one if needed)